soh/soh/Enhancements/randomizer/Plandomizer.cpp

```diff
@@ -0,0 +1,966 @@
+    { RG_BUY_BOMBS_10,              		"ITEM_BOMB" },
+    { RG_BUY_BOMBS_20,              		"ITEM_BOMB" },
+    { RG_DEKU_NUTS_5,               		"ITEM_NUT" },
+    { RG_DEKU_NUTS_10,              		"ITEM_NUT" },
+    { RG_BUY_DEKU_NUTS_5,           		"ITEM_NUT" },
+    { RG_BOMBCHU_5,                 		"ITEM_BOMBCHU" },
+    { RG_BOMBCHU_10,                		"ITEM_BOMBCHU" },
+    { RG_BOMBCHU_20,                		"ITEM_BOMBCHU" },
+    { RG_ARROWS_5,                  		"ITEM_ARROWS_SMALL" },
+    { RG_ARROWS_10,                 		"ITEM_ARROWS_MEDIUM" },
+    { RG_ARROWS_30,                 		"ITEM_ARROWS_LARGE" },
+    { RG_TREASURE_GAME_HEART,       		"ITEM_HEART_PIECE" },
+    { RG_DEKU_SEEDS_30,             		"ITEM_SEEDS" },
+    { RG_SOLD_OUT,                  		"ITEM_SOLD_OUT" },
+    { RG_TRIFORCE,                  		"ITEM_TRIFORCE" },
+};
+
+std::vector<std::string> hintPool = {
+    "They say that %gGanondorf's Mom%w is going out with %ySqueak%w!",
+	"They say that %gProxySaw%w is still fixing %yCaladius's Bugs%w...",
+	"They say that %gItsHeckinPat%w is still just %yEyeballing it%w.",
+	"They say that %gCaladius%w is working on %yV2%w of something.",
+	"They say that %gdice%w is a funny name for a %ytaco%w.",
+	"They say %g2Ship Rando%w is still blocked by %yV3%w...",
+	"They say if you click your heels and say %gframebuffer%w 3 times, %yArchez%w appears!",
+	"They say %gVB%w stands for %yVirtual Bananas%w... Probably.",
+	"They say %gZeru%w is still routing his %yHundo%w.",
+	"They say %gRaccoonCloud%w is still looking for his %yHover Boots%w.",
+	"They say %gItsHeckinPat%w foreclosed on his %yHut%w.",
+	"They say %gRaccoonCloud%w is part of the %yInner Circle%w.",
+	"They say %gMoonlitxShadows%w is the %rleader%w of the %yDork Army%w.",
+	"They say %gGanondorf%w hates the %yInternet%w.",
+	"They say %gMido's House%w hoards %yTrash%w.",
+	"They say %gSweettalking Ganondorf%w rewards %yHis Heart%w.",
+	"They say %gaMannus%w said %yGo To Bed%w!",
+	"They say %gCaladius%w is a %yPinhead%w!",
+	"They say %gRaccoonCloud%w loves the %yIce Cavern%w.",
+	"They say %gNo One%w should forget %yHover Scrub%w!",
+	"They say %gMoonlitxShadows%w likes to %ySlide%w.",
+	"They say that %gRaccoonCloud%w is still learning %yB1 Skip%w...",
+	"They say that %gBackwalking%w should be %rBanned%w!",
+	"They say that %gGoron's%w should always have %yLong Necks%w!",
+	"They say that %gCaladius%w has a %ytendency to lose his shirt%w!",
+	"They say that if your %rSkip keeps Failing%w, you're probably an %yESS Off%w.",
+	"They say that %gLogic%w is just a %ySuggestion%w.",
+	"They say there's %gAlways Logic%w in %yNo Logic%w...",
+	"They said that %rFredomato%w has just %yone more push up%w to do!"
+};
+
+Rando::Item plandomizerRandoRetrieveItem(RandomizerGet randoGetItem) {
+    auto randoGetItemEntry = Rando::StaticData::RetrieveItem(randoGetItem);
+    return randoGetItemEntry;
+}
+
+ImVec4 plandomizerGetItemTint(Rando::Item randoItem) {
```

Comment:
I would rename this stuff to references to itemColor rather than tint. Tint isn't as clear, imo.